### DISCUSSION TITLE
Handbrake arguments passed on, if skip transcode is True?

### DISCUSSION THREAD

**Discussion by drlauridsen** [automatic-ripping-machine/automatic-ripping-machine#872] [Q&A]
EDIT: Disregard this - I just realized that I have misunderstood the flow completely.. 
It is MakeMKV that does the ripping and converting to MKV - so of course the HB arguments are not passed on..
I will have to find out if makeMKV can accept arguments.
For now I have used the tool mkvmerge to remove audiotracks..
---

I have disabled transcoding which works great, but I have a lot of imported DVDs, where I want to skip all other audiotracks than english and danish, and the same for subtitles.
Mostly because it is a waste of disk space, but also because the default audio and subtitle language will be a language I understand :-)

I have tried with the following argument under HB_ARGS_DVD:

--native-language eng --native-dub eng --subtitle-lang-list eng,dan --all-subtitles

But all other languages were still included in the mkv file, both audio and subtitles
Is this because I have disabled transcoding?

**Comment by 1337-server** [comment]
For MakeMKV you need to use its own profiles

Info here:
https://forum.makemkv.com/forum/viewtopic.php?f=10&t=4386
https://forum.makemkv.com/forum/viewtopic.php?t=4385
